src/strongbox/core.clj

```diff
@@ -733,15 +733,16 @@
          ;; todo: metrics gathering is good, but this is a little adhoc.
          ;; some metrics we'll emit for the user
          [num-installed num-matched] [(count installed-addon-list) (count matched)]
-         unmatched-names (set (map :name unmatched))]
+         ;; we don't bother trying to match ignored addons, we shouldn't bother reporting they were also unmatched.
```

Comment:
"we don't match ignored addons, we shouldn't report we couldn't find them either"